pkg/queue/health/health_state_test.go

```diff
@@ -67,79 +67,58 @@ func TestHealthStateHealthHandler(t *testing.T) {
 		alive        bool
 		shuttingDown bool
 		prober       func() bool
-		isAggressive bool
 		wantStatus   int
 		wantBody     string
 	}{{
-		name:         "alive: true, K-Probe",
-		alive:        true,
-		isAggressive: false,
-		wantStatus:   http.StatusOK,
-		wantBody:     queue.Name,
+		name:       "alive: true, K-Probe",
+		alive:      true,
+		wantStatus: http.StatusOK,
+		wantBody:   queue.Name,
 	}, {
-		name:         "alive: false, prober: true, K-Probe",
-		prober:       func() bool { return true },
-		isAggressive: false,
-		wantStatus:   http.StatusOK,
-		wantBody:     queue.Name,
+		name:       "alive: false, prober: true, K-Probe",
+		prober:     func() bool { return true },
+		wantStatus: http.StatusOK,
+		wantBody:   queue.Name,
 	}, {
-		name:         "alive: false, prober: false, K-Probe",
-		prober:       func() bool { return false },
-		isAggressive: false,
-		wantStatus:   http.StatusServiceUnavailable,
+		name:       "alive: false, prober: false, K-Probe",
+		prober:     func() bool { return false },
+		wantStatus: http.StatusServiceUnavailable,
 	}, {
-		name:         "alive: false, no prober, K-Probe",
-		isAggressive: false,
-		wantStatus:   http.StatusOK,
-		wantBody:     queue.Name,
+		name:       "alive: false, no prober, K-Probe",
```

Comment:
we can drop these `K-Probe` mentions anymore - since I believe those were for the aggressive state